presto-docs/src/main/sphinx/release/release-348.md

```diff
@@ -0,0 +1,62 @@
+# Release 348 (XX Dec 2020)
+
+## General Changes
+
+* Add support for `DISTINCT` clause in aggregations within correlated subqueries. ({issue}`5904`)
+* Improve query performance by reducing worker to worker communication overhead. ({issue}`6126`)
+* Reduce memory pressure and improve performance of queries that contain join. ({issue}`6176`)
+* Fix ``EXPLAIN ANALYZE`` for certain queries that contain broadcast join. ({issue}`6115`)
+* Fix planning failures for queries that contain filtered aggregations and outer joins. ({issue}`6141`)
+* Fix incorrect results when correlated subquery in join contains aggregation functions such as `array_agg` or `checksum`. ({issue}`6145`)
+* Fix duplicate call to event listener on query completion, if query failed early during preparation. ({issue}`6103`)
+
+## Web UI
+
+* Add support for OAuth2 authorization in Web UI. ({issue}`5355`)
+* Fix invalid operator stats reporting in stage performance view. ({issue}`6114`)
+
+## JDBC Driver Changes
+
+* Allow reading `timestamp with time zone` value as `ZonedDateTime` using `ResultSet.getObject(int column, Class<?> type)` method. ({issue}`307`)
+* Change representation of a `row` value. `ResultSet.getObject` now returns an instance of `io.prestosql.jdbc.Row` class, which better represents
+  the returned value. Previously a `row` value was represented as a `Map` instance, with unnamed fields being named like `field0`, `field1`, etc. 
+  You can access the previous behavior by invoking `getObject(column, Map.class)` on the `ResultSet` object. ({issue}`4588`)
+* Fix failure when reading a `timestamp` or `timestamp with time zone` value with second fraction greater than or equal to 999999999500 picoseconds. ({issue}`6147`)
```

Comment:
```suggestion
* Fix failure when reading a `timestamp` or `timestamp with time zone` value with a second fraction greater than or equal to 999999999500 picoseconds. ({issue}`6147`)
```
I am not sure though

Reply:
Actually, "with seconds fraction" sounds better.